### PR TITLE
add: Button Component

### PR DESCRIPTION
**Pull request by Varu98**

### CODE REVIEW

File: styles/buttons.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+@import url(./main.css);
+/* ------------- */
+/* Buttons */
+/* ------------- */
+
+.primary-btn {
+  background-color: hsl(var(--bee-yellow));
+  padding: 0.7rem 1rem;
+  margin: 1rem 1rem;
```
Comment (gopalbbharadva) [inline styles/buttons.css:10]: Suggestion: margin:1rem will affect the same.

**Review by gopalbbharadva — APPROVED** [review]
Vardan what you can do is,  introduce about what the PR, about how much variant you made of component, some description about your PR eventually.

File: styles/buttons.css [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  color: hsl(var(--dark-blue));
+  font-weight: 900;
+  border: none;
+  border-radius: 5px;
+  box-shadow: 0 5px 5px rgba(0, 0, 0, 0.3);
+  outline: none;
```
Comment (kiranpratyush) [inline styles/buttons.css:16]: let's review to not  remove outline  because of accessibility reason

File: styles/buttons.css [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+.primary-btn:hover {
+  transform: scale(1.1);
+  transition: 0.3s;
+}
+
+/* Secondary Button */
+.secondary-btn {
+  background-color: hsl(var(--light-blue));
+  padding: 0.7rem 1rem;
+  margin: 1rem 1rem;
```
Comment (kiranpratyush) [inline styles/buttons.css:27]: could we change it to margin:1rem

File: styles/buttons.css [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+  color: hsl(var(--white) / 2);
+  font-weight: 900;
+  border: none;
+  border-radius: 5px;
+  box-shadow: 0 5px 5px rgba(0, 0, 0, 0.3);
+  outline: none;
```
Comment (kiranpratyush) [inline styles/buttons.css:33]: Let's review to not change outline

File: styles/buttons.css [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+  color: hsl(var(--dark-blue));
+}
+
+/* Outline Button */
+.outline-btn {
+  color: hsl(var(--dark-blue));
+  font-weight: 900;
+  background: none;
+  border: none;
+  outline: none;
```
Comment (kiranpratyush) [inline styles/buttons.css:89]: Outline

File: styles/buttons.css [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+  border-radius: 50%;
+  border: 3px solid hsl(var(--dark-blue));
+  box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.3);
+  background-color: hsl(var(--light-blue));
+}
+
+/* icon  Button */
+.btn-icon {
+  display: flex;
+  flex-direction: row;
```
Comment (kiranpratyush) [inline styles/buttons.css:117]: flex-direction is row by default

**Review by kiranpratyush — APPROVED** [review]

File: styles/buttons.css (hunk 1 above)
Comment (Varu98) [reply styles/buttons.css:10]: right 😆 thanks!

**Comment by Varu98** [comment]
> Vardan what you can do is, introduce about what the PR, about how much variant you made of component, some description about your PR eventually.

Great! thanks for the feedback will keep in mind next time

**Pull request merged**